pkg/ast/ast.go

```diff
@@ -1272,10 +1272,24 @@ type ColumnDef struct {
 
 	Null token.Pos // position of "NULL"
 
-	Name    *Ident
-	Type    SchemaType
-	NotNull bool
-	Options *ColumnDefOptions // optional
+	Name          *Ident
+	Type          SchemaType
+	NotNull       bool
+	GeneratedExpr *GeneratedColumnExpr // optional
```

Comment:
Please update the above `ColumnDef` doc comment by adding `{{ GeneratedExpr | sqlOpt }}`.